app/styles/newcomponents/chart-bar.scss

```diff
@@ -0,0 +1,9 @@
+.c3-axis-x > .tick {
+  fill: none; // remove axis labels from x-axis
+}
+
+.c3 path,
+.c3 line {
+  fill: none;
+  stroke: #ddd;
```

Comment:
All colors should be declared in variables.scss and referenced only by `$color` variables in other scss files.  There are already a few different grey options there - will one of those work?